compiler/rustc_errors/src/styled_buffer.rs

```diff
@@ -49,29 +66,31 @@ impl StyledBuffer {
     }
 
     fn ensure_lines(&mut self, line: usize) {
-        while line >= self.text.len() {
-            self.text.push(vec![]);
-            self.styles.push(vec![]);
+        while line >= self.lines.len() {
+            self.lines.push(vec![]);
         }
     }
 
+    /// Sets `chr` with `style` for given `line`, `col`.
+    /// If line not exist in `StyledBuffer`, adds lines up to given
+    /// and fills last line with spaces and `Style::NoStyle` style
```

Comment:
```suggestion
    /// If `line` does not exist in our buffer, adds empty lines up to the given
    /// and fills the last line with unstyled whitespace.
```
maybe something like this reads slightly better :thinking:

Reply:
Fixed.